simul.spec

```diff
@@ -48,10 +47,10 @@ and coherence of parallel filesystems.
 %if %{with_mpich}
 %package mpich
 Summary: Simul for MPICH
-%if "%{_host_vendor}" == "suse"
-BuildRequires: mpich-devel%{?_isa} lua-lmod libfabric-devel glibc
+%if (0%{?suse_version} > 0)
+BuildRequires: mpich-devel%{?_isa} lua-lmod libfabric-devel
 %endif
-%if "%{_host_vendor}" == "redhat"
+%if (0%{?rhel} > 0)
```

Comment:
Per the previous comment, remove this line:
```suggestion
```
and line 55 so that `BR: mpich-devel%{?_isa}` is common for all distros.